pkix/cert_auth.go

```diff
@@ -61,6 +61,8 @@ var (
 		IsCA: true,
 		// Not allow any non-self-issued intermediate CA
 		MaxPathLen: 0,
+		// Must be true if MaxPathLen==0
```

Comment:
Looks like `MaxPathLenZero: true` is sufficient and `MaxPathLen` will default to zero is unset.